docs/init.md

```diff
@@ -0,0 +1,98 @@
+# Usage Instructions
+
+`wasm-pack` is a tool designed to help with packaging binaries including wasm,
+and publishing them on npm.
+
+## Prerequisites
+
+You should have Rust and npm installed already. You can confirm
+that these are installed using the following commands:
+
+```sh
+# Check if Rust is installed.
+cargo --version
+
+# Check if npm is installed.
+npm --version
+```
+
+You can find more information about installing Rust
+[here][rust-wasm-install-info], and more information about installing npm
+[here][npm-install-info].
+
+You will need an account on npm to upload a package. The sign up page
+can be found [here][npm-signup-info].
+
+[rust-wasm-install-info]: https://rust-lang-nursery.github.io/rust-wasm/setup.html
+[npm-install-info]: https://www.npmjs.com/get-npm
+[npm-signup-info]: https://www.npmjs.com/signup
+
+## wasm-pack Installation
+
+You can install `wasm-pack` using the following command:
+
+```sh
+cargo install wasm-pack
```

Comment:
As an aside, do you think it would be worth adding a `wasm-pack update` command that would wrap this? I can open an issue/PR and get that done if it sounds like a good idea to you all.